### DISCUSSION TITLE
Testers

### DISCUSSION THREAD

**Discussion by saladuit** [codewars/codewars.com#2713] [Q&A]
I'm doing these kata's to learn more about TDD (Test Driven Development), but I'm really missing the option to see the testers that are being used for the exercises that I'm doing. Is it possible to see the test that is behind the example tests after completing a kata? Thank you!

**Comment by Blind4Basics** [comment]
Most of the availble information is either [here](https://docs.codewars.com/languages) or [there](https://github.com/codewars/runner)

**Comment by hobovsky** [comment]
After completing a kata, you can see full test suite on its Solutions or Discourse tab, hidden on a collapsible panel:

![image](https://user-images.githubusercontent.com/23709795/172073662-00e77b54-d086-468e-a42f-0c5083268bc7.png)

**Reply by Blind4Basics** [reply]
oh, yeah, that surely was the actual question... 'x)

**Reply by saladuit** [reply]
Thank you soo much! Great work on the website!
